src/components/Comment/Comment.tsx

```diff
@@ -82,12 +83,41 @@ const iconWrapper = css`
   white-space: nowrap;
 `;
 
+const timestampWrapperStyles = css`
+  margin-left: 10px;
+  display: flex;
+  flex-direction: column;
+  justify-content: center;
+`;
+
 export const avatar = (avatarSize: number): string => css`
   border-radius: ${avatarSize + 10}px;
   width: ${avatarSize}px;
   height: ${avatarSize}px;
 `;
 
+const Column = ({ children }: { children: JSX.Element | JSX.Element[] }) => (
```

Comment:
Here we go again :) Probably can refactor other areas to use this as a util in another PR.

Reply:
Ha, yeah, I had that thought. I added it here to make the html more readable but it's really only useful where you have identical css, which isn't that often.

Aside: It would be more useful if we went with the `const PinkColumn = styled(Column)'color: pink; '` pattern so we can express what I want via composition.